frontend/src/routes/settings.tsx

```diff
@@ -16,7 +16,7 @@ function SettingsScreen() {
 
   const saasNavItems = [
     { to: "/settings/user", text: t("SETTINGS$NAV_USER") },
-    { to: "/settings/git", text: t("SETTINGS$NAV_GIT") },
+    { to: "/settings/integrations", text: t("SETTINGS$NAV_GIT") },
```

Comment:
@openhands please rename the translation key `SETTINGS$NAV_GIT` to be `SETTINGS$NAV_INTEGRATIONS`